src/compiler/scala/tools/nsc/backend/jvm/LabelNode1.java

```diff
@@ -0,0 +1,15 @@
+package scala.tools.nsc.backend.jvm;
+
+import scala.tools.asm.Label;
+import scala.tools.asm.tree.LabelNode;
+
+public class LabelNode1 extends LabelNode {
+    public LabelNode1() {
+    }
+
+    public LabelNode1(Label label) {
+        super(label);
+    }
+
+    public int flags;
+}
```

Comment:
The ASM maintainers said they didn't want to make `Label.flags` public. I think we can achieve the same result by using this subclass of `LabelNode` with a spot for some user-data.